Makefile

```diff
@@ -37,6 +37,6 @@ clean:
 setup:
 	@echo "\n📥 Downloading and extracting runtime...\n"
 	mkdir -p /tmp/runtime
-	wget -O /tmp/blockless-runtime.tar.gz https://github.com/blocklessnetwork/runtime/releases/download/v0.3.1/blockless-runtime.linux-latest.x86_64.tar.gz
+	wget -O /tmp/blockless-runtime.tar.gz https://github.com/blocklessnetwork/runtime/releases/download/v0.3.4/blockless-runtime.darwin-latest.x86_64.tar.gz
```

Comment:
not sure if the `darwin` is explicit (since im assuming we cant run this in a container this way)

Reply:
yeah this make file doesn't build the container image, just a buff for local development.